---a/net/minecraft/item/EnumAction.java
+++b/net/minecraft/item/EnumAction.java
@@ -1,7 +1,6 @@
 package net.minecraft.item;
 
-public enum EnumAction
-{
+public enum EnumAction {
     NONE,
     EAT,
     DRINK,
@@ -9,4 +9,4 @@
     BOW;
 
     private static final String __OBFID = "CL_00000073";
-}
+}
